<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWnd</class>
 <widget class="QWidget" name="UserWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserWnd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="cb_target"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="MsgView" name="msgView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QWidget" name="msgEdit" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>msgView</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MsgView</class>
   <extends>QWidget</extends>
   <header>msgview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>msgEdit</sender>
   <signal>messageSendRequested(QString)</signal>
   <receiver>UserWnd</receiver>
   <slot>messageSendRequested(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>messageSendRequested(QString)</slot>
 </slots>
</ui>
